Makefile

```diff
@@ -154,6 +154,10 @@ build-egen: build-dir
 test-pdp: cover-out
 	$(AT)/pdp && $(GOTESTRACE)
 
+.PHONY: test-pdp-integration
```

Comment:
I don't see a need of the integration tests it the form they have been added here. Themis package includes several different things - pdp, parsers, server, client libraries. What do the tests integrate? Why only some aspects of numbers, but not whole thing? I see those more like parser tests so please move them to related parser directory.

Reply:
Maybe **integration** is a poor choice of word, but these are definitely not **unit** tests either. They don't just test the parsing, but the actual calculation too.

On your other point, I agree they definitely don't test *all* aspects of numbers. Please let me know what are the significant omissions and I'll definitely add those too.

Reply:
I prefer to test calculations inside pdp constructing policies with Make* functions. Basically, we don't need much integration inside single repository.

Reply:
OK. Let me give it a shot.